Scripts/Interfaces/IFilterUI.cs

```diff
@@ -12,6 +12,25 @@ public interface IFilterUI
 		/// Set accessor for the filter list
 		/// </summary>
 		List<string> filterList { set; }
+		string searchQuery { get; }
+	}
+
+	public static class FilterUIExtensionMethods
+	{
+		public static bool TestFilter(this IFilterUI filterUI, string type)
```

Comment:
Not sure what Test means here. Probably should be named IsFiltered, which means the return value should be reversed.